rotkehlchen/premium/premium.py

```diff
@@ -46,9 +49,30 @@ class RemoteMetadata(NamedTuple):
     data_size: int
 
 
+class UserLimits(TypedDict):
+    """User limits for rotki premium subscription."""
+    # Maximum number of devices that can be registered with the premium account
+    limit_of_devices: int
+    # Maximum number of profit and loss events that can be processed
+    pnl_events_limit: int
+    # Maximum size in megabytes for database backups stored on rotki servers
+    max_backup_size_mb: int
+    # Maximum number of history events that can be stored and processed
+    history_events_limit: int
+    # Maximum number of report lookups that can be performed
+    reports_lookup_limit: int
+
+
 COMPONENTS_VERSION: Final = 14
 DEFAULT_ERROR_MSG: Final = 'Failed to contact rotki server. Check logs for more details'
-DEFAULT_OK_CODES: Final = (HTTPStatus.OK, HTTPStatus.UNAUTHORIZED, HTTPStatus.BAD_REQUEST)
+DEFAULT_OK_CODES: Final = (
+    HTTPStatus.OK,
+    HTTPStatus.UNAUTHORIZED,
+    HTTPStatus.BAD_REQUEST,
+    HTTPStatus.CREATED,
+)
+NEST_API_ENDPOINTS: Final = ('backup', 'devices', 'limits')
+USER_LIMIT_TYPE = Literal['history_events_limit', 'pnl_events_limit', 'reports_lookup_limit']
```

Comment:
rather than literals would it be better to have them as a normal enum? It is better with linters and autocompletions